src/multi_physics/QED_tests/validation/validation.ipynb

```diff
@@ -290,7 +290,7 @@
     "\n",
     "def BW_d2N_dt_dchi(chi_phot, gamma_phot, chi_ele):    \n",
     "    coeff_BW = fine_structure * me*c**2/hbar \n",
-    "    return coeff_BW*BW_F(chi_phot, chi_ele)*(gamma_phot/gamma_phot)\n",
+    "    return coeff_BW*BW_F(chi_phot, chi_ele)*(chi_phot/gamma_phot)\n",
```

Comment:
This is the only real change. All the other changes are simply a consequence of re-running the script.